src/rust/engine/options/src/config_tests.rs

```diff
@@ -55,49 +48,6 @@ fn test_display() {
     );
 }
 
-#[test]
-fn test_multiple_sources() {
```

Comment:
Multiple configs is no longer a thing to be tested separately within config_tests.rs, it is amply tested along with multiple sources in general, in tests.rs.